.github/scale-config.yml

```
@@ -63,6 +63,12 @@ runner_types:
     is_ephemeral: false
     max_available: 400
     os: linux
+  linux.g4.12xlarge.nvidia.gpu:
```

```suggestion
  linux.g4dn.12xlarge.nvidia.gpu:
```

Let's change the name to be representative of what the instance type is